### PR TITLE
chore: add ember tests for migrate init

### PR DESCRIPTION
**Pull request by brendaHuang**
Adding ember scenario tests for `migrate init`.

### CODE REVIEW

**Comment by lynchbomb** [comment]
Looks like you have some failing linting

**Review by wondersloth — APPROVED** [review]
Looks good. This should close out the `migrate init` portion of ember apps and addons in cli.

File: packages/cli/test/commands/migrate/init-command.ember-addon.test.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+          lintConfig,
+          lintConfigDefault,
+          tsConfig,
+          tscLintScript,
+        } = await runDefault(app.dir);
+
+        expect(cleanOutput(stdout, app.dir)).toMatchSnapshot();
+
+        for (const devDep of REQUIRED_DEPENDENCIES) {
+          expect(Object.keys(devDeps!).includes(devDep));
```
[the remark this answers is not in the document]
Comment (brendaHuang) [reply packages/cli/test/commands/migrate/init-command.ember-addon.test.ts:40]: will change this in next PR

[the remark this answers is not in the document]
Comment (wondersloth) [reply packages/cli/test/commands/migrate/init-command.ember-addon.test.ts:40]: No need to change this @brendaHuang I just had to figure out what the syntax meant. Very cool as it ensures that devDeps is not undefined or null

**Pull request merged**